internal/scorecard/alpha/tests/bundle_test.go

```diff
@@ -117,3 +118,26 @@ func TestBasicAndOLM(t *testing.T) {
 
 	}
 }
+
+func TestOwnedCRDsForResources(t *testing.T) {
+	ownedCRDDescription := operators.CRDDescription{
```

Comment:
I think we need to check 2 scenarios:
a) when has the owner should not fail
2) when the owner is missing should fail 

Also, I think would be nice to follow its standard. See the test in the file [here](https://github.com/camilamacedo86/operator-sdk/blob/master/pkg/ansible/paramconv/paramconv_test.go#L92). Since they are small I think it would be a good example to illustrate the suggestion.

Reply:
Since we were checking the pass condition in `TestBasicAndOLM` had skipped it here. Have added both the conditions now, please have a look at it.

Reply:
solved